### ISSUE TITLE
This plugin is incompatible with phonegap-plugin-push plugin because android-support-v4.jar is included (cordova build android failing)

### ISSUE THREAD

**Issue by jacqueskang** [Wizcorp/phonegap-facebook-plugin#1183]
If both phonegap-facebook-plugin and phonegap-plugin-push are installed in the same project then cordova build android will fail.
Investigation shows that this is because android-support-v4.jar is included in this plugin.
See https://github.com/phonegap/phonegap-plugin-push/issues/338

Is there a way to fix this or provide any walkaround please?
Thanks in advance.

**Comment by tutcugil** [comment]
i have same issue, cant work this plugin with phonegap-plugin-push because of push uses support 13 this plugin uses support v4

any solution for this?

**Comment by bau720123** [comment]
hello guys
don't waste your time in "phonegap-facebook-plugin"
please follow this
https://github.com/phonegap/phonegap-plugin-push#co-existing-with-facebook-plugin

**Comment by Clebiez** [comment]
Hi, I found a solution for this problem after many time spend on Google :p
You must use cordova 5.0 (or equivalent in phonegap) for this solution.

Found the plugin.xml in plugins/com.phonegap..plugins.facebookConnect and replace the line 60 
```
<framework src="platforms/android/FacebookLib" custom="true" />
```

by them :

```
<framework src="com.android.support:support-v4:+" />
<framework src="com.facebook.android:facebook-android-sdk:3.23.0" />
```

You will replace a local dependence by a graddle dependence more efficient and without conflict

**Comment by boten** [comment]
hi 
i replaced the line but getting the same error ?
do i need to refresh my android project somehow ?

**Comment by Clebiez** [comment]
Yes sorry, you must rm the platform and add it again to consider this change.

**Comment by boten** [comment]
the build is working again :)

thank you :)

**Comment by programmersujeet** [comment]
@Clebiez , Just implemented your hack. Yes its working, amazing

**Comment by mribbons** [comment]
I also encountered this issue, I am also using another [plugin](https://github.com/EddyVerbruggen/cordova-plugin-googleplus) which may be causing the conflict.

The fix identified by @Clebiez works.

Can this be integrated into the repo?

I would be happy to set up a PR but I don't know enough about cordova to understand the ramifications of the change.

**Comment by dgavriil** [comment]
Clebiez you are a god!!!
